src/DB-api/environments/migrations/create_db.sql

```diff
@@ -0,0 +1,102 @@
+
+CREATE TABLE `animals` (
+  `id` int PRIMARY KEY NOT NULL,
+  `organization_id` varchar(255) NOT NULL,
+  `type_id` int,
+  `species` varchar(255),
+  `primary_breed_id` int,
+  `secondary_breed_id` int,
+  `is_mixed_breed` boolean,
+  `is_unknown_breed` boolean,
+  `primary_color_id` int,
+  `secondary_color_id` int,
+  `tertiary_color_id` int,
+  `age` varchar(255),
+  `gender_id` int,
+  `size` varchar(255),
+  `coat_id` int,
+  `is_spayed_neutered` boolean,
+  `is_house_trained` boolean,
+  `is_declawed` boolean,
+  `is_special_needs` boolean,
+  `is_shots_current` boolean,
+  `is_environment_children` boolean,
+  `is_environment_dogs` boolean,
+  `is_environment_cats` boolean,
+  `name` varchar(255) NOT NULL,
+  `description` varchar(255) NOT NULL,
+  `status` varchar(255) NOT NULL,
+  `published_at` timestamp NOT NULL
+);
+
+CREATE TABLE `animal_tags` (
+  `animal_id` int,
+  `tag_descriptor` varchar(255)
+);
+
+CREATE TABLE `animal_types` (
+  `id` int PRIMARY KEY NOT NULL,
+  `type_id` int NOT NULL
+);
```

Comment:
this is an enum table,  so the id is the type_id and the type_id should really be the string(varchar) of the type.